bootstrap.yml

```diff
@@ -514,6 +515,29 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
 
+  - name: radare2
+    source:
+      subdir: '3rdparty'
+      url: 'https://codeload.github.com/radareorg/radare2/tar.gz/refs/tags/5.4.0'
+      format: 'tar.gz'
+      extract_path: 'radare2-5.4.0'
```

Comment:
Usually after this line goes `patch-path-strip: 3` which is needed for the patching script to do its magic. It is good use to add it even if the patch is empty, like for other packages.